GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -100,6 +100,7 @@ public static bool TrySetRequiredGitConfigSettings(Enlistment enlistment)
                 { "index.version", "4" },
                 { "merge.stat", "false" },
                 { "merge.renames", "false" },
+                { "pack.useBitmaps", "false" },

```

Comment:
This change is incorrectly shown by GitHub, after I picked changes in upstream:master to my fork and rebased mac_installer_prototype branch on to master.

Use this URL to view the correct diff.

Reply:
It looks like you did a rebase, and then did a merge before pushing, so you have some duplicate commits in this PR and that is messing up the diff. You can fix this by dropping the latest merge commit, and force pushing the results of your rebased topic branch.

Reply:
I had a messed up git commit graph. Also there was a Github issue with identifying the right merge-bases which @derrickstolee investigated (turns out Github support is already aware of the issue).